mmdet/models/dense_heads/anchor_head.py

```diff
@@ -620,70 +618,89 @@ def _get_bboxes_single(self,
         """
         cfg = self.test_cfg if cfg is None else cfg
         assert len(cls_score_list) == len(bbox_pred_list) == len(mlvl_anchors)
+        batch_size = cls_score_list[0].shape[0]
         # convert to tensor to keep tracing
         nms_pre_tensor = torch.tensor(
```

Comment:
Seems `cls_score_list[0].new_tensor` would be ok. Seems that `dtype=torch.long` is not necessary ?

`